docs/api/useTable.md

```diff
@@ -95,7 +95,7 @@ The following options are supported on any column object you can pass to `column
   - Receives the table instance and column model as props
   - Must either be a **string or return valid JSX**
   - If a function/component is passed, it will be used for formatting the header value, eg. You can use a `Header` function to dynamically format the header using any table or column state.
-- `Cell: Function | React.Component => JSX`
+- `Cell: Function | React.Component => JSX` <a id="column-object-cell"></a>
```

Comment:
What is this for?

Reply:
This produces the anchor that is referenced by `[Cell](#column-object-cell)`. I always find it helpful to have cross-references as clickable links..

If you know of a more markdown-esque syntax to produce this, I would be more than happy to know of it.